hack/bookstore_client.py

```diff
@@ -1,4 +1,5 @@
-#!/usr/bin/python -u
```

Comment:
Good catch; i'm adding stuff to the boilerplate checker to be permissive on python shebangs.